src/js/index.js

```diff
@@ -0,0 +1,102 @@
+import { XApp } from "./xapp/index.js";
+import { ServerMenuAPI } from "./api.js";
+
+function createApp() {
+    const api = new ServerMenuAPI();
+
+    const app = new XApp({
+        el: document.getElementById("app"),
+    });
+
+    app.setData({
+        categories: [],
+        categorySelected: {
+            name: "",
+            key: "",
+        },
+        menuList: [],
+        menuInput: "",
+        menuCount: 0,
+    });
+
+    app.setMethods({
+        async createMenu(e) {
+            e.preventDefault();
+            if (this.data.menuInput !== "") {
```

Comment:
값이 빈 값이 아닐 경우라고 조건문이 작성되어있는데 값이 있을경우로 작성해도 되지 않을까 싶습니다 .. !!
```suggestion
            if (this.data.menuInput) {
```